packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -193,6 +194,11 @@ type DefaultProps = {
     PerseusInteractiveGraphRubric
 > satisfies PropsFor<typeof InteractiveGraph>;
 
+0 as any as WidgetProps<
+    InteractiveGraphPublicWidgetOptions,
+    PerseusInteractiveGraphRubric
+> satisfies PropsFor<typeof InteractiveGraph>;
```

Comment:
This is probably the most important line of code in this PR. It ensures that answerless data can be assigned to the widget props.